client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -365,11 +365,21 @@ export default React.createClass( {
 
 	recordSubmit() {
 		const errors = formState.getErrorMessages( this.state.form );
-		analytics.tracks.recordEvent( 'calypso_contact_information_form_submit', {
-			errors,
+		let tracksEventObject = {
 			errors_count: errors && errors.length || 0,
 			submission_count: this.state.submissionCount + 1
-		} );
+		};
+
+		tracksEventObject = reduce(
+			formState.getErrorMessages( this.state.form ),
+			function( result, value, key ) {
```

Comment:
Super minor, but it would be a bit more consistent with the syntax of this module to use a fat arrow function here.